### PR TITLE
Structopt to 0.3

### PR DESCRIPTION
**Pull request by danielSanchezQ**
Fixes #1708

### CODE REVIEW

File: jormungandr-integration-tests/src/common/jcli_wrapper/certificate/commands.rs [hunk 1]
```diff
@@ -26,6 +26,9 @@ impl CertificateCommands {
             .arg("new")
             .arg("stake-delegation")
             .arg(&delegation_id)
+            // This was added as part of updating structopt to 0.3+, it changed the interface
+            // and required to add `--` before the stake pool ids
+            .arg("--")
```
Comment (danielSanchezQ) [inline jormungandr-integration-tests/src/common/jcli_wrapper/certificate/commands.rs]: I had to add this in order to make the command work. I could not find the change in the documentation of `structopt` but it is due to a change in it. It means that now a command should be run with:
```
jcli.exe certificate new stake-delegation delegation_id -- stake_pool_id
```
I'm taking a look to see if I can make it work without the `--` to keep the same behavior as in previous versions

File: jormungandr-integration-tests/src/jcli/key/from_bytes.rs [hunk 2]
```diff
@@ -52,6 +52,6 @@ pub fn test_from_bytes_for_unknown_key() {
     jcli_wrapper::assert_key_from_bytes_fails(
         &byte_key_file,
         "ed25519Exten",
-        "Invalid value for '--type <key_type>'",
+        "Invalid value for '--type <key-type>':",
```
Comment (danielSanchezQ) [inline jormungandr-integration-tests/src/jcli/key/from_bytes.rs]: Command renaming behavior [changed here](https://github.com/TeXitoi/structopt/blob/master/CHANGELOG.md#change-default-case-from-verbatim-into-kebab-by-0ndorio-202)

Comment (NicolasDP) [reply jormungandr-integration-tests/src/jcli/key/from_bytes.rs]: this one is not a problem, it does not change the user's APIs

File: jormungandr-integration-tests/src/common/jcli_wrapper/certificate/commands.rs (hunk 1 above)
Comment (vincenthz) [reply jormungandr-integration-tests/src/common/jcli_wrapper/certificate/commands.rs]: is that maybe a bug in structopt ?

Comment (eugene-babichenko) [reply jormungandr-integration-tests/src/common/jcli_wrapper/certificate/commands.rs]: Strange. I tested on macOS with and without `--` and both work just fine.

Comment (danielSanchezQ) [reply jormungandr-integration-tests/src/common/jcli_wrapper/certificate/commands.rs]: @eugene-babichenko , did you check with this version (`0.3^`)?

Comment (danielSanchezQ) [reply jormungandr-integration-tests/src/common/jcli_wrapper/certificate/commands.rs]: I made a `cargo clean`, `cargo build -all` and `cargo test`, it is working as expected. I may had a bad build when testing it before.

Comment (eugene-babichenko) [reply jormungandr-integration-tests/src/common/jcli_wrapper/certificate/commands.rs]: Great to hear. Yes, I was playing with this branch.

**Comment by danielSanchezQ** [comment]
I removed the `--`, tests are working. The issue was triggered because of a bad local build.

**Pull request merged**